src/views/Team/index.view.tsx

```diff
@@ -1,8 +1,35 @@
 import * as React from "react";
+import MLHBanner from "components/MLHBanner/index.view";
+import Navbar from "components/Navbar/index.view";
+import Hero from "./components/Hero/index.view";
+import Card from "./components/Card/index.view";
+import Footer from "./components/Footer/index.view";
+
+import { navbarProps } from "views/Home/props/navbar.js";
+import { members } from "./props/members";
+
 import "./Team.scss";
 
+const description =
+  "Meet our team of creators, thinkers, strategists, and believers, collaborating together to create something amazing. ";
 const TeamPageView: React.FC = () => {
-  return <div className="TeamPage">Hello from TeamPageView</div>;
+  return (
+    <>
+      <MLHBanner />
+      <div className="TeamPage">
+        <Navbar page={"Team"} {...navbarProps} />
```

Comment:
```suggestion
      <Navbar page={"Team"} {...navbarProps} />
      <MLHBanner />
      <div className="TeamPage"
```
This shouldn't make a difference but this would help replicate the same structure found in `Home`.

Reply:
You right! I thought we needed this to be outside of the container, but looks like it doesn't really matter.